app/components/container/containerController.js

```diff
@@ -48,38 +48,26 @@ function ($q, $scope, $state, $transition$, $filter, Container, ContainerCommit,
     });
   };
 
-  $scope.start = function () {
-    $('#loadingViewSpinner').show();
-    Container.start({id: $scope.container.Id}, {}, function (d) {
-      update();
-      Notifications.success('Container started', $transition$.params().id);
-    }, function (e) {
-      update();
-      Notifications.error('Failure', e, 'Unable to start container');
-    });
-  };
+  function pastTense(v) {
```

Comment:
I do not like this function. Messages should not be build via functions, they should be display entirely (that will make things easier for i18n later on).